extending-valtimo/plugin/custom-plugin-definition.md

```diff
@@ -6,6 +6,23 @@ Valtimo offers the functionality needed to create and add plugins to Valtimo imp
 
 ### Creating a plugin definition
 
+#### Dependencies
+
+To create a custom plugin in your project, the following dependencies are needed:
+- valtimo:plugin
+- valtimo:documenten-api
```

Comment:
Only `valtimo:plugin`  is necessary. The others are not.

Reply:
@marijnritense actually Theo and I found out we need them all.

Reply:
Do you have the exceptions? Assuming what you're saying is true, the current state of plugins is unacceptable...

Reply:
Smartdocuments works without these dependencies by the way, and contains the smartdocuments plugin.